clusters/development/overlay/third-party/snyk-monitor/helmRelease.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: velero
```

Comment:
Probably not correct name and namespace?

Reply:
thank you, fixed.